### PR TITLE
test for direct git push

### PR DESCRIPTION
**Pull request by timocafe**
I am testing if I have the right for pull request ....

### CODE REVIEW

**Comment by timocafe** [comment]
Yes I can

**Comment by eile** [comment]
You can, but you should not. PRs need review.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ RELEASE:
 The serial test unroll 4 fails, again for the division. It is very wierd
 as the operator /= call the system division and not Newton-Raphson
 
+Release 1.3.0
+
+- New lisense lgpl 3.0
```
Comment (eile) [inline README.md:48]: q.e.d.:

Typo: License. Name is LGPL (it's an abbreviation)

File: README.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ RELEASE:
...
+- Open Source Standard
```
Comment (eile) [inline README.md:49]: What does that mean, in particular to a user?

**Comment by timocafe** [comment]
Hello,

Fabien said me this morning I have the right to push, I just verify an making some tests on text files
Next commit/pull request will be verify by a member of the team.

Best,

Tim

**Comment by eile** [comment]
Good :)

**Pull request merged**